nntrainer/graph/network_graph.cpp

```diff
@@ -719,6 +719,7 @@ int NetworkGraph::initialize() {
      */
     if (!is_input_node(lnode)) {
       auto &input_layers = lnode->getInputLayers();
+      lnode->resizeInputDimensions(input_layers.size());
```

Comment:
We are not concerning when the input_layers have more than one outputs.

Reply:
Yes. we don't have/support such case for now.